cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -145,16 +161,36 @@ class ScannerCommand : CliktCommand(name = "scan", help = "Run external license
                 "times. For example: --label distribution=external"
     ).associate()
 
-    private val scannerFactory by option(
-        "--scanner", "-s",
-        help = "The scanner to use, one of ${Scanner.ALL}."
-    ).convertToScanner().default(ScanCode.Factory())
-
-    private val projectScannerFactory by option(
-        "--project-scanner",
-        help = "The scanner to use for scanning the source code of projects. By default, projects and packages are " +
-                "scanned with the same scanner as specified by '--scanner'."
-    ).convertToScanner()
+    private val scannerOption by mutuallyExclusiveOptions(
+        option(
+            "--scanner", "-s",
+            help = "The scanner to use, one of ${Scanner.ALL}."
+        ).convertToScanner(),
+        option(
+            "--experimental-scanners",
+            help = "A comma-separated list of experimental scanners to use. This option is mutually exclusive with " +
+                    "'--scanner'. The experimental scanner implementation scans by provenance instead of by package. " +
+                    "This improves reuse of stored scan results and increases performance if multiple packages are " +
+                    "coming from the same source code repository. The experimental scanner is work in progress and " +
+                    "it is therefore not recommended to use it in production.\n" +
+                    "Possible values are: ${ScannerWrapper.ALL}"
+        ).convertToScannerWrapperFactories()
+    ).single().default(ScannerOption.Stable(ScanCode.Factory()))
+
+    private val projectScannerOption by mutuallyExclusiveOptions(
+        option(
+            "--project-scanner",
+            help = "The scanner to use for scanning the source code of projects. By default, projects and packages " +
+                    "are scanned with the same scanner as specified by '--scanner'."
```

Comment:
Nit: While at it, should we also add the "Possible values are" phrase here? Or drop it from below. I didn't add the list of scanners here initially as this option already refers to the general option which does have the list.

Reply:
I decided to be more verbose and list the values for each option, to make it easier to understand for the user.